src/OpenOfficeConvert/OOStyles.cs

```diff
@@ -2219,14 +2212,18 @@ private void CreateLeftGuideword(int i)
                     }
                     break;
             }
-            _writer.WriteStartElement("text:span");
-            _writer.WriteAttributeString("text:style-name", "MT1");
-            _writer.WriteStartElement("text:variable-get");
-            _writer.WriteAttributeString("text:name", "Left_Guideword_L");
-            _writer.WriteEndElement(); //text:variable-get
-            _writer.WriteEndElement();
 
-            if (_projInfo.ProjectInputType.ToLower() == "dictionary")
+	        if (!(isMirrored && i == 19))

```

Comment:
Is this i == 19 related to setting the minimum height to 19 above? Perhaps we need an explanation of what is happening and why this should only be done for mirrored pages with i == 19.

Reply:
The "i == 19" defines the position of the guidewords, "19" position mean to Mirrored page with RightGuideword. In that position, we don't need to insert LeftGuideword.